cmd/cli/main.go

```diff
@@ -0,0 +1,361 @@
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	v1 "github.com/k8up-io/k8up/v2/api/v1"
+	"github.com/k8up-io/k8up/v2/cli/restore"
+	"github.com/k8up-io/k8up/v2/cmd"
+	"github.com/urfave/cli/v2"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	Command = &cli.Command{
+		Name:        "cli",
+		Description: "CLI commands that can be executed everywhere, currently just restore is supported",
+		Subcommands: []*cli.Command{
+			{
+				Name:   "restore",
+				Action: RunRestore,
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Destination: &restore.Cfg.Snapshot,
+						Required:    true,
+						Name:        "snapshot",
+						Usage:       "Required ; ID of the snapshot `kubectl get snapshots`, set, via cli or via env: ",
```

Comment:
Setting this to `latest` by default could be an UX improvement.

Reply:
I disagree - it can be nice feature but ONLY if users are 100% sure what they do, otherwise they can mess with their apps while just playing arround with cli